### ISSUE TITLE
cjsx doesn't get react-hot

### ISSUE THREAD

**Issue by jasonrose** [HenrikJoreteg/hjs-webpack#123]
```js
if (isInstalled('react-hot-loader') && config.spec.devServer.hot) {
      config.module.loaders[0].loaders.unshift('react-hot')
    }
```

only adds `react-hot` to the babel loader.  I need to add it to the cjsx and cs loaders too.  I'm done for the day, but I plan on somehow tackling this tomorrow.

**Comment by jasonrose** [comment]
```
loaders [ { test: /\.(js|jsx|babel)$/,
    exclude: /node_modules/,
    loaders: [ 'react-hot', 'babel-loader' ] },
  { test: /\.coffee$/, loaders: [ 'coffee-loader' ] },
  { test: /\.cjsx$/,
    loaders: [ 'coffee-loader', 'cjsx-loader' ] },
  { test: /\.json$/, loaders: [ 'json' ] },
  { test: /\.otf(\?\S*)?$/, loader: 'url-loader?limit=1' },
  { test: /\.eot(\?\S*)?$/, loader: 'url-loader?limit=1' },
  { test: /\.svg(\?\S*)?$/,
    loader: 'url-loader?mimetype=image/svg+xml&limit=1' },
  { test: /\.ttf(\?\S*)?$/,
    loader: 'url-loader?mimetype=application/octet-stream&limit=1' },
  { test: /\.woff2?(\?\S*)?$/,
    loader: 'url-loader?mimetype=application/font-woff&limit=1' },
  { test: /\.(jpe?g|png|gif)$/, loader: 'url-loader?limit=1' },
  { test: /\.css$/,
    loader: 'style-loader!css-loader!postcss-loader' },
  { test: /\.styl$/,
    loader: 'style-loader!css-loader!postcss-loader!stylus-loader' },
  { test: /\.less$/,
    loader: 'style-loader!css-loader!postcss-loader!less-loader' } ]
new entry [ 'webpack-dev-server/client?https://localhost:8000',
  'webpack/hot/only-dev-server',
  '/Users/jrose/development/git/shutterfly/robot/app/components/host/host.coffee' ]
new entry [ 'webpack-dev-server/client?https://localhost:8000',
  'webpack/hot/only-dev-server',
  '/Users/jrose/development/git/shutterfly/robot/app/components/main/app.coffee' ]
new entry [ 'webpack-dev-server/client?https://localhost:8000',
  'webpack/hot/only-dev-server',
  '/Users/jrose/development/git/shutterfly/robot/app/components/integration/integration.coffee' ]
```

Is what I get when I log out cfg.loaders in my webpack.config.js right now.

**Comment by lukekarrys** [comment]
Closes by #124
